lib/jellyfish/config_reader.ex

```diff
@@ -75,6 +76,30 @@ defmodule Jellyfish.ConfigReader do
         _other ->
           fallback.(value)
       end
+    else
+      nil
+    end
```

Comment:
You can omit this `else` block, and because other functions don't put it, let's be consistent with them

Reply:
I added this `else` and `spec` because [we forgot that `read_boolean` doesn't always return a boolean.](https://github.com/jellyfish-dev/jellyfish/pull/109#discussion_r1373499739). But I can remove them if you prefer.

Reply:
Yeah, let's remove them